Updated the introductory material to use POI-CM or (#79)
opengeospatial/poi: 22-053/sections/00-preface.adoc

[[abstract_section]]
== Abstract

> POI-CM is an open conceptual data model for representing information about points of interest (POI). It is defined through a Unified Modeling Language (UML) object model. This UML model extends the ISO Technical Committee 211 (TC211) conceptual model standards for spatial and temporal data. Building on the ISO fundation assures that the features described in the POI Models share the same spatial-temporal universe as features described by related standards (e.g., CityGML).

Reviewer: Should Model be capitalized in "POI Models"?

```suggestion
POI-CM is an open conceptual data model for representing information about points of interest (POI). It is defined through a Unified Modeling Language (UML) object model. This UML model extends the ISO Technical Committee 211 (TC211) conceptual model standards for spatial and temporal data. Building on the ISO foundation assures that the features described in the POI Models share the same spatial-temporal universe as described by related standards (e.g., CityGML).
```

> The aim of the development of the OGC POI standard is to reach a common definition of the basic entities, attributes, and relations of "points of interest". In the most broad terms, a point of interest is a location about which information of general interest is available. A POI can be as simple as a set of coordinates and an identifier, or more complex such as a three dimensional model of a building with names in various languages, information about open and closed hours, and a civic address

Reviewer: The definition of POI needs to appear in the terminology definitions clause (it looks like that may need to be added?) and, if one exists, a source for the definition needs to be cited in the definition and populated in the bibliography. 

```suggestion
The aim of developing the OGC POI conceptual model is to reach a common definition of the basic entities, attributes, and relations of "points of interest." In the broadest terms, a point of interest is a location about which information of general interest is available. A POI can be as simple as a set of coordinates and an identifier, or more complex such as a three-dimensional model of a building with names in various languages, information about open and closed hours, and a civic address.
```

Reviewer: You can create a new issue with the suggestion to create a terms/definitions clause or a glossary, or point to the clause in the draft standard, and then it will kick this comment down the road for future task.

> This Users Guide provides extended explanations and examples for the individual concepts that are defined in the POI Conceptual Model Standard. Both documents, the Conceptual Model Standard and the Users Guide, are mutually linked to facilitate navigation between corresponding sections in these documents.

Reviewer: ```suggestion
This Users Guide provides extended explanations and examples for the individual concepts that are defined in the POI Conceptual Model Standard. Both the Conceptual Model Standard and this Users Guide are mutually linked to facilitate navigation between corresponding sections in these documents.
```
